modules/iscsi/data/org.freedesktop.UDisks2.iscsi.xml

```diff
@@ -82,7 +82,12 @@
         @options: Additional options.
         @since: 2.1.3
 
-        Sets a new iSCSI initiator name.
+        Sets a new iSCSI initiator name. This is typically done by populating the
+        <filename>/etc/iscsi/initiatorname.iscsi</filename> file.
```

Comment:
Maybe iscsid could watch the file and restart/reload itself?

Reply:
> Maybe iscsid could watch the file and restart/reload itself or just read the new value or is that too much to ask?

Oh, forgot to add that this would be racy so we'll need an explicit call anyway.

Reply:
> Also, this is important for `blivet` as well. In most cases the `iscsid.service` is not running by default and is autostarted (by socket connection), reading the changed `InitiatorName`.

Note to myself: not a problem. https://github.com/storaged-project/blivet/blob/3.6-devel/blivet/iscsi.py#L215